src/GeositeFramework/js/Legend.js

```diff
@@ -62,6 +62,10 @@ define(['use!Geosite',
         },
 
         getLayerTemplate: function(legend, service, layer) {
+            // exclude some layers from legend
+            if ( layer.name.startsWith("_") ){ 
```

Comment:
Functionality looks fine, but the implementation with `startsWith` will throw runtime errors in IE where it's not supported in any version (just Edge).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith#Browser_compatibility